src/style.css

```diff
@@ -0,0 +1,127 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --link-color: #00acdc;
+  --border-color: #f3f3f3;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
> Лучше не сбрасывать margin и padding для всех тегов - нагружает страницу

It might be slow but now it's ok to write down like this.
**Check it out** -- [Why you don't like asterisk](https://levelup.gitconnected.com/the-css-universal-selector-is-slow-and-should-be-avoided-yet-another-lie-eb4d15f4c320)

Reply:
Цікава стаття, але знову ж таки це думка одного розробника, їх може бути багато :)

Reply:
@solaryasha то якщо я розділяю цю думку, то хіба не можу так писати?)